app/components/viral/dropdown_component.rb

```diff
@@ -4,24 +4,34 @@ module Viral
   # Dropdown component
   class DropdownComponent < Viral::Component
     renders_many :items, Dropdown::ItemComponent
-    attr_reader :distance, :dropdown_styles, :label, :icon_name, :caret, :skidding, :trigger, :tooltip
+    attr_reader :distance, :dropdown_styles, :label, :icon_name, :caret, :skidding, :trigger, :tooltip,
+                :caret_icon, :selected
 
     TRIGGER_DEFAULT = :click
     TRIGGER_MAPPINGS = {
       click: 'click',
       hover: 'hover'
     }.freeze
 
+    CARET_ICON_MAPPINGS = {
+      bottom: 'chevron_down',
+      left: 'chevron_left',
+      right: 'chevron_right',
+      top: 'chevron_up'
+    }.freeze
+
     # rubocop:disable Metrics/ParameterLists
     def initialize(label: nil, tooltip: '', icon: nil, caret: false, trigger: TRIGGER_DEFAULT, skidding: 0,
-                   distance: 10, dropdown_styles: '', **system_arguments)
+                   distance: 10, dropdown_position: :bottom, dropdown_styles: '', selected: false, **system_arguments)
```

Comment:
**nitpick:** Can you add LookBook previews for the new dropdown positions?

Reply:
Added in 6112b6d4e0f11f5d06200f9d07d4df67fbfd1bcd

Reply:
Looks good, thanks